src/Symfony/Component/Ldap/Ldap.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Ldap;
+
+use Symfony\Component\Ldap\Adapter\AdapterInterface;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ * @author Francis Besset <francis.besset@gmail.com>
+ * @author Charles Sarrazin <charles@sarraz.in>
+ */
+final class Ldap implements LdapInterface
+{
+    private $adapter;
+
+    /**
+     * Constructor.
+     *
+     * @param AdapterInterface $driver A configured Ldap adapter
+     */
```

Comment:
The whole phpdoc can be removed as it does not add anything we don't already have with the function signature.